build.sbt

```diff
@@ -307,11 +300,22 @@ lazy val bench = project
   .settings(Settings.bench)
 
 lazy val layer = project
-  .dependsOn(raster)
+  .dependsOn(raster, `raster-testkit` % Test)
   .settings(commonSettings)
   .settings(Settings.layer)
 
 lazy val store = project
   .dependsOn(layer)
   .settings(commonSettings)
   .settings(Settings.store)
+
+lazy val gdal = project
+  .dependsOn(raster, `raster-testkit` % Test)
+  .settings(commonSettings)
+  .settings(Settings.gdal)
+
+lazy val `gdal-spark` = project
```

Comment:
what is the better name / project structure for tests that include tiling, spark and GDAL?